internal/ad/backends/winbind/winbind_test.go

```diff
@@ -32,8 +32,10 @@ func TestWinbind(t *testing.T) {
 		wbclientBehavior string
 		staticADDomain   string
 		staticADServer   string
+		hostname         string
```

Comment:
You don’t have a test case and don’t assign hostname to anything in any test (which is a valid test to add btw :)). So, they all end up with ubuntu.